.gitignore

```
@@ -9,3 +9,4 @@ storybook-static
 .nyc_output
 *.tsbuildinfo
 /cypress/
+.idea
```

We're missing a trailing newline here. It's not super critical, but helps for future edits. 

Ref: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3#

```suggestion
.idea

```

---

Added!